en/docs/install-and-setup/upgrading-wso2-api-manager/upgrading-guidelines.md

```diff
@@ -3,9 +3,13 @@
 This section contains the complete upgrading process related to the WSO2 API Manager.
 Go through the guidelines given below before attempting to upgrade the production environment.
 
-!!! note
-    To upgrade **from a version older than 1.8.0**, follow the instructions in the document that was released immediately after your current release and upgrade incrementally.
+!!! note "If you are migrating from an older API-M version"
+    - To upgrade **from a version older than 1.8.0**, follow the instructions in the document that was released immediately after your current release and upgrade incrementally.
+    - To upgrade **from a 2.x version older than 2.6.0**, follow the instructions in [2.6.0 migration guide](https://docs.wso2.com/display/AM260/Upgrading+from+the+Previous+Release) and migrate to API-M 2.6.0 first. Then refer to [API-M 2.6.0 to 4.1.0 Migration Guide]({{base_path}}/install-and-setup/upgrading-wso2-api-manager/260-to-410/upgrading-from-260-to-410/) to migrate to API-M 4.1.0.
```

Comment:
To upgrade **from a  WSO2 API-M 2.x version older than 2.6.0 or  WSO2 API-M 1.10/1.8.0/1.9.0/1.9.1**,